schemas/public-transport-feed.schema.json

```
@@ -0,0 +1,129 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://region-registry.skedgo.com/public-transport-feed.schema.json",
+  "title": "Public Transport Data Feed",
+  "description": "Public Transport Data Feed specification.",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "Unique identifier for this feed. Recommended format is ISO Country Code + Provider name; for example au-tfnsw. For multi country feeds, recommended format is Continent Code + Provider name; for example eu-flixbus.",
+      "type": "string",
+      "pattern": "^[a-z]{2}-(([a-z]|-|_)+)$"
+    },  
+    "name": {
+      "description": "Human friendly name for this feed.",
+      "type": "string",
+      "minLength": 1,
+      "pattern": "^(([A-Z]|[a-z]|-|_)+)$"
```

See below. I'd remove the pattern.

```suggestion
      "minLength": 1
```